Dockerfile-linux.template

```
@@ -1,7 +1,7 @@
 FROM placeholder
 
 # add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
-RUN groupadd -r mongodb && useradd -r -g mongodb mongodb
+RUN groupadd -r mongodb && useradd -m -r -g mongodb mongodb
```

Rather than creating a home directory for a system account, we should probably just set it somewhere that `mongodb` can write, like `/tmp`.

Like this:
```suggestion
RUN groupadd -r mongodb && useradd --home-dir /tmp -r -g mongodb mongodb
```

Or this?
(fully expand the arguments for clarity; adjust now-edited `RUN` line to match semicolon formatting of other lines)
```suggestion
RUN set -eux; \
       groupadd --system mongodb; \
       useradd --system --gid mongodb --home-dir /tmp mongodb
```

Maybe even add `--gid` and `--uid` with explicit number `999` (the current value) like https://github.com/docker-library/postgres/pull/93 to `groupadd` and `useradd`, respectively.

---

Or maybe we just add `ENV HOME /tmp` so that any UID that the user runs as will work.

---

How about a home-dir of `/var/mongodb`, similar to other system accounts?

```
> docker run mongo cut -d: -f 6,1 /etc/passwd | sort -t : -k 2
bin:/bin
proxy:/bin
sync:/bin
sys:/dev
mongodb:/home/mongodb
_apt:/nonexistent
nobody:/nonexistent
root:/root
games:/usr/games
daemon:/usr/sbin
backup:/var/backups
man:/var/cache/man
gnats:/var/lib/gnats
list:/var/list
mail:/var/mail
irc:/var/run/ircd
lp:/var/spool/lpd
news:/var/spool/news
uucp:/var/spool/uucp
www-data:/var/www
```

---

Still need to create the home directory to address the original issue.
The mongodb user ID should not share a global HOME env var with any other accounts.